draft-ietf-ppm-dap.md

```diff
@@ -1364,22 +1367,28 @@ following checks:
       find it helpful to track additional information, like the timestamp, so
       that the storage used for anti-replay can be sharded efficiently.
 
-1. Check that a report is only aggregated in a batch if it was included in all
-   previous collections for the batch. Some VDAFs allow a batch to be collected
-   multiple times; the aggregators must check that the exact same set of reports
-   is being aggregated each time. Checking that no reports are omitted
-   from a batch is covered in {{batch-validation}}.
-   If this check fails, the input share MUST be marked as invalid with the error
-   `batch_collected`. This prevents collectors from learning anything about
+1. If the report pertains to a batch that was previously collected, then make
+   sure the report was already included in all previous collections for the
+   batch. If not, the input share MUST be marked as invalid with error
+   "batch_collected". This prevents Collectors from learning anything about
    small numbers of reports that are uploaded between two collections of a
    batch.
-   The helper must consider a batch as collected when it has sent the
-   `AggregateShare` to the leader.
-   [TODO: If a section to clarify report and batch states is added this can be
-   removed. See Issue #384]
+
+    * Implementation note: The Leader considers a batch to be collected once it
+      has completed a collection job for a CollectReq message from the
+      Collector; the Helper considers a batch to be collected once it has
+      responded to an AggregateShareReq message from the Leader. A batches is
+      determined by query ({{query}}) conveyed in these messages. Queries must
+      satisfy the criteria covered in {{batch-validation}}. These criteria are
+      meant to restrict queries in a way make it easy to determine wither a
+      report pertains to a batch that was collected.
+
+      [TODO: If a section to clarify report and batch states is added this can be
+      removed. See Issue #384]
```

Comment:
Reviewer note: I have tried to rework with this chunk so that the actual thing to check is mentioned in the first sentence. @simon-friedberger please double check that I haven't inadvertently changed the meaning here.